sdk/storage/azure-storage-blob-changefeed/CHANGELOG.md

```diff
@@ -2,7 +2,6 @@
 
 ## 12.0.0-beta.9 (2021-09-15)
 ### Other changes
-#### Dependency updates
```

Comment:
I'd remove Other changes as well..

Reply:
This was the change that I got approved by the Eng Sys team. Since we're going to evaluate our changelog formatting for the next release, I'd rather leave this at what they agreed on to wrap up this release.